sip/sip.go

```diff
@@ -208,8 +215,9 @@ func (v *DispatchRuleValidator) Validate(r *livekit.SIPDispatchRuleInfo) error {
 				if v.opt.AllowConflicts {
 					continue
 				}
-				return twirp.NewErrorf(twirp.InvalidArgument, "Conflicting SIP Dispatch Rules: same Trunk+Number+PIN combination for for %q and %q",
-					printID(r.SipDispatchRuleId), printID(r2.SipDispatchRuleId))
+				return twirp.NewErrorf(twirp.InvalidArgument,
+					"Dispatch rule for the same trunk, number, and pin combination already exists in dispatch rule %q %q",
```

Comment:
Could we capitalize pin as `PIN`?

```suggestion
					"Dispatch rule for the same trunk, number, and PIN combination already exists in dispatch rule %q %q",
```